lab-kayla/test/server-test.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+// const router = require('../lib/router');
```

Comment:
**TODO FOR LOST POINTS**

Take out commented code. If you have a test module in progress, just let us know in the Canvas comment that you purposely haven't set it up yet.